### ISSUE TITLE
Consider changing the loading screen

### ISSUE THREAD

**Issue by nvs** [nvs/gem#167]
If would be great to start a contest to select a winner. Or at the very least, get a few contributions from people. Using a new loading screen has the distinct advantage of shrinking the file size. Also, it would help rebrand the map.

**Comment by nvs** [comment]
Is there a loading screen with text that shows player load progress?

**Comment by GaDaHo** [comment]
Good Idea for at some point maybe Bryvnx would have remodeled the loading screen but if changed I know you won't forget to mentioned his name.
